kafka-ui-api/src/main/java/com/provectus/kafka/ui/config/auth/LdapSecurityConfig.java

```diff
@@ -99,8 +99,10 @@ public BaseLdapPathContextSource contextSource() {
   @Primary
   public LdapAuthoritiesPopulator ldapAuthoritiesPopulator(BaseLdapPathContextSource contextSource) {
     var authoritiesPopulator = new DefaultLdapAuthoritiesPopulator(contextSource, props.getGroupFilterSearchBase());
+    authoritiesPopulator.setGroupSearchFilter(props.getGroupFilterSearchFilter());
```

Comment:
according to DefaultLdapAuthoritiesPopulator's  sources default value for search filter is `(member={0})` - if we don't provide this via config explicitly- it will be null, which will cause default behavior change?

Reply:
Thanks, fixed!